Assets/MixedRealityToolkit.Providers/WindowsMixedReality/WindowsMixedRealityCameraSettings.cs

```diff
@@ -3,13 +3,16 @@
 
 using Microsoft.MixedReality.Toolkit.CameraSystem;
 using Microsoft.MixedReality.Toolkit.Utilities;
+using System.Collections.Generic;
 
 #if UNITY_WSA
 using UnityEngine.XR.WSA;
 #endif // UNITY_WSA
 
 #if WINDOWS_UWP
-using Windows.Graphics.Holographic;
+using Windows.Foundation.Metadata;
+#elif UNITY_WSA && DOTNETWINRT_PRESENT
+using Microsoft.Windows.Foundation.Metadata;
 #endif // WINDOWS_UWP
```

Comment:
These `#ifdef`s look like they will exclude the feature when running on-device, which is ironically the only time it will be relevant. The `Microsoft.Windows` namespace should work fine in both environments, so really there should just be an `#if` for `DOTNETWINRT_PRESENT` with no `#elif`.

Reply:
I may be mistaken about the meanings of `WINDOWS_UWP` and `UNITY_WSA` so please correct me if I'm wrong.

Reply:
BTW, you can use `Microsoft.Windows` and `global::Windows` side-by-side if needed, though any gap in `Microsoft.Windows` is potentially something we could add so you can use it uniformly (within reason - I don't plan to extend this to include `Windows.Storage` for example).

Reply:
> meanings of WINDOWS_UWP and UNITY_WSA

I'll get the definitions of these from the unity docs and add here. We can discuss / test once i have them posted.

Reply:
From Unity:

UNITY_WSA | #define directive for Universal Windows Platform. Additionally, NETFX_CORE is defined when compiling C# files against .NET Core and using .NET scripting backendA framework that powers scripting in Unity. Unity supports three different scripting backends depending on target platform: Mono, .NET and IL2CPP. Universal Windows Platform, however, supports only two: .NET and IL2CPP. More infoSee in Glossary.

Reply:
WINDOWS_UWP is not set in Unity. It is set when compiling in Visual Studio

Reply:
I am more than happy to take time to analyze the #if directives together to ensure they are correct.